apps/omg_utils/lib/omg_utils/http_rpc/client_adapter.ex

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
```

Comment:
same comment as above. this is a really wasteful implementation considering we have full swagger specifications we could utilize. that's what they're for!

Reply:
let's take it to that other comment's thread (and let's be constructive about it. I actually don't know whether swagger specs were in place, and decided they're staying, when this `Client` was first added, so this still existing might be a case of inertia. No need to judge, if we can improve this, we will.)